lightning/src/ln/channel.rs

```diff
@@ -4504,7 +4522,15 @@ impl<Signer: Sign> Channel<Signer> {
 		}
 	}
 
-	pub fn get_accept_channel(&self) -> msgs::AcceptChannel {
+	pub fn inbound_is_awaiting_accept(&self) -> bool {
+		self.inbound_awaiting_accept
+	}
+
+	pub fn set_is_awaiting_accept(&mut self) {
```

Comment:
This is generally fine with me, actually, but we should make this `#[cfg(test)]`. An alternative way to accomplish the same test would be to split `get_accept_channel` into an utility to get the message itself and another pub method to get the message while setting the awaiting flag.